exp-player/addon/components/exp-player.js

```diff
@@ -148,6 +148,9 @@ able to continue the study.
                 this._transition();
                 this.set('frameIndex', frameIndex + 1);
                 this.set('framePage', 0);
+                this.get('session').set('frameIndex', frameIndex + 1);
```

Comment:
I am getting the value of `frameIndex` from the session but `session.frameIndex` does not automatically get updated when the value of the `frameIndex` changes: https://github.com/CenterForOpenScience/isp/pull/68/files#diff-c5ef69c69ee1fddc9d65f6d3878873dfR8